components/globe.js

```diff
@@ -84,7 +84,7 @@ class WebGLGlobe extends React.Component {
 
     if (webGLEnabled) {
       return (
-        <StyledGlobe innerRef={(el) => { this.globeRef = el }} />
+        <StyledGlobe ref={(el) => { this.globeRef = el }} />
```

Comment:
It's cool that they have fixed this. Do you know if this requires a particular version of React (I seem to recall this wasn't possible in previous styled-components version due to react limitations)? 

Be sure to do some testing of this (especially when used in form/input components) as there may be some breakage.

Reply:
This is based on the `forwardRef` feature coming in with [`react@16.3.0`](https://reactjs.org/blog/2018/03/29/react-v-16-3.html#forwardref-api). Which means we have to upgrade react too which seems to be a non-breaking upgrade. We should expedite writing tests.